code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -33,7 +33,7 @@
 		if(9)
 			new /obj/item/organ/brain/alien(src)
 		if(10)
-			new /obj/item/organ/heart/cursed(src)
+			new /obj/item/grant_omnitongue(src)
```

Comment:
I honestly thought the same thing when i saw the PR, put a picture of the leaning tower of pisa on the front or something